workitem_blackbox_test.go

```diff
@@ -130,3 +135,96 @@ func TestListByFields(t *testing.T) {
 
 	test.DeleteWorkitemOK(t, nil, nil, controller, wi.ID)
 }
+
+func TestUnauthorizeCreateWI(t *testing.T) {
+	resource.Require(t, resource.Database)
+	// Build A token
+	token := jwt.New(jwt.SigningMethodRS256)
+	privatekey, err := jwt.ParseRSAPrivateKeyFromPEM(([]byte(RSAPrivateKeyTest)))
+	if err != nil {
+		t.Fatal("Could not parse Key ", err)
+	}
+	publickey, err := jwt.ParseRSAPublicKeyFromPEM(([]byte(RSAPublicKeyTest)))
+	if err != nil {
+		t.Fatal("Could not parse Key ", err)
+	}
+	tokenStr, err := token.SignedString(privatekey)
+	if err != nil {
+		t.Fatal("Could not sign the token ", err)
+	}
+
+	// Build a request
+	req, err := http.NewRequest("POST", "/api/workitems", nil)
+	if err != nil {
+		t.Fatal("could not create a HTTP request")
+	}
+
+	// Add Authorization Header in wrong format
+	authToken := fmt.Sprintf("WRONG TOKEN Format Bearer %s", tokenStr)
```

Comment:
Yeah, working on it. Sounds like TableDriven Testing possible here. let me check :)